lib/Sema/TypeCheckAttr.cpp

```diff
@@ -2380,8 +2384,17 @@ void AttributeChecker::visitDifferentiableAttr(DifferentiableAttr *attr) {
     for (unsigned i : indices(parsedWrtParams)) {
       auto paramLoc = parsedWrtParams[i].getLoc();
       switch (parsedWrtParams[i].getKind()) {
-      case ParsedAutoDiffParameter::Kind::Index: {
-        unsigned index = parsedWrtParams[i].getIndex();
+      case ParsedAutoDiffParameter::Kind::Named: {
+        auto nameIter = std::find(originalParamNames.begin(),
```

Comment:
Use `llvm::find` with `originalParamNames` as range.

Reply:
Done in https://github.com/apple/swift/pull/22000/commits/aec0072e4eb43e7c59c6b73d980030413deaceea.